pkg/file/file.go

```diff
@@ -104,26 +106,17 @@ func AtomicWrite(path string, data []byte, mode os.FileMode) (err error) {
 }
 
 func Exists(name string) bool {
```

Comment:
We should probably get rid of this function, but I'm not trying to refactor everything in this patch. I prefer to handle filesystem errors as they come up in the operations we're trying to do, since the stat call could fail for any number of reasons unrelated to the file's existence.